src/pages/Encounters/ReportBuilder/index.tsx

```diff
@@ -69,9 +71,45 @@ export default function ReportBuilderList({
           <div className="grid grid-cols-1 md:grid-cols-2 gap-4 mt-2">
             <CardListSkeleton count={5} />
           </div>
-        ) : reportTemplateData?.results?.length === 0 ? (
-          <div className="text-center text-gray-500 p-5">
-            {t("no_templates_found")}
+        ) : !reportTemplateData?.results ||
+          reportTemplateData?.results?.length === 0 ? (
+          <div className="flex flex-col items-center justify-center p-10 border border-dashed border-gray-300 rounded-lg bg-gray-50 my-4">
+            <div className="text-center space-y-5 max-w-md">
+              <div className="mx-auto bg-white p-4 rounded-full w-16 h-16 flex items-center justify-center border border-gray-200 shadow-sm">
+                <CareIcon
+                  icon="l-file-medical"
+                  className="h-8 w-8 text-blue-500 transform scale-170"
```

Comment:
you can control `CareIcon`'s icon sizes using `text-*` size classes instead

```suggestion
                  className="text-blue-500 text-lg"
```

Reply:
Thank you for the suggestion. I have implemented the changes.